### ISSUE TITLE
emacs-bin-master.tar.gz not found?

### ISSUE THREAD

**Issue by tsuu32** [npostavs/emacs-travis#3]
Thanks for such a great CI for emacs package!

I found EMACS_VERSION=master build always failed.
Please see https://travis-ci.org/tsuu32/quelpa-use-package/builds/659732059?utm_medium=notification&utm_source=github_status.

I think Currently emacs-bin-master.tar.gz not found.

and followning travis.yml in README
```
env:
  matrix:
    - EMACS_VERSION=23.4
    - EMACS_VERSION=24.5
    - EMACS_VERSION=25.3
    - EMACS_VERSION=master
  allow_failures:
    - env: EMACS_VERSION=master
```
should be 
```
env:
  jobs:
    - EMACS_VERSION=23.4
    - EMACS_VERSION=24.5
    - EMACS_VERSION=25.3
    - EMACS_VERSION=master

jobs:
  allow_failures:
    - env: EMACS_VERSION=master
```

**Comment by npostavs** [comment]
> I found EMACS_VERSION=master build always failed.

Hmm, looks like there was an upload error, but the script still deleted the old master version even though it's not supposed to in that case. https://travis-ci.org/npostavs/emacs-travis/jobs/659722642
I added some print statements which will hopefully give some more clues.

> and followning travis.yml in README

Examples on the web seem to do it your way, so I guess it's correct. Updated.
